Remove empty hyperlink on Prerequisites header

juvchan opened the pull request:
The hyperlink on the Prerequisites links to nowhere.
Propose to remove the hyperlink from this header text

azurecla commented:
Hi __@chrisbenti__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@chrisbenti Thanks for the contribution!

Commit:
Remove empty hyperlink on Prerequisites header

The hyperlink on the Prerequisites links to nowhere.
Propose to remove the hyperlink from this header text
